README.md

```diff
@@ -64,6 +64,14 @@ The only required argument is the directoy where all SCSS files are located. Run
 
 Watch for files that get added or are changed and generate the corresponding type definitions.
 
+### `--skipInitial`
```

Comment:
Thoughts on using `ignoreInitial` to keep the API naming consistent with `chokadir`? Looks like it's providing the same thing functionally so would be nice to keep consistent across libraries.

Let me know your thoughts or if you had another reason 👌

Reply:
I'm good either way – I only did skipInitial to be the same as node-sass-chokidar (but changing the case to match the other options in this package). Want me to change it?

Reply:
Interesting 🤔 if you wouldn't mind updating to be consistent with `chokadir` (`ignoreInitial`) that'd be great. Curious why `node-sass-chokidar` chose to deviate.

Reply:
Okay! I'll get that updated right now.

Reply:
Done